Makefile

```diff
@@ -25,14 +30,37 @@ generate: $(GOBINDATA_BIN)
 $(GOBINDATA_BIN):
 	go get -u github.com/jteeuwen/go-bindata/...
 
-test:
+test:	verify
 	go test ./...
 
 test-integration:
 	go test -v -tags integration ./test/integration
 
+verify:	verify-gofmt
```

Comment:
is this formatting error a subtle joke?

Reply:
hmm, something weird there - it shows up as tab in vi.  Something expanding it on github?

Reply:
Are you suggesting a tab there was *intentional*? I suspect we'd have a lively argument over a `make fmt` command... 😁

Reply:
lol